pkgs/tools/misc/phoronix-test-suite/tests.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, phoronix-test-suite }:
+
+with phoronix-test-suite;
```

Comment:
It is usually discouraged because it makes it obscures where variables come from. It is not hard to figure out in small expressions like this one but `let inherit (phoronix-test-suite) pname version;` might be still preferable to not give `with` any more ground.

Reply:
ok, i understand. the inherit is good, will use it.

reminds me of pythons `from phoronix-test-suite import pname version`